examples/tokens/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Repo, getDefault, EcdsaKey, ChainTree, establishTokenTransaction, mintTokenTransaction, sendTokenTransaction, receiveTokenTransactionFromPayload } from 'tupelo-wasm-sdk';
```

Comment:
totally outside the scope of this pr, but I think `getDefault()` is a vague function name.

Reply:
I thought the same. I wanted to import the module and use via that instead, but it wouldn't work since the default export of tupelo-wasm-sdk lacked the right symbols.

Reply:
agree... I'll add it as a static method on Community